src/app/pages/credentials/new-users/all-users/user-details/user-profile-card/user-profile-card.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-card class="card">
+  <mat-card-header>
+    <h3 mat-card-title>
+      {{ 'Profile' | translate }}
+    </h3>
+  </mat-card-header>
+
+  <mat-card-content>
+    @if (user(); as user) {
```

Comment:
Since `user()` input is required there shouldn't be a situation when this condition is not true.